tests/net/socket/reuseaddr/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Please change the commit subject of "net: tests: add tests for SO_REUSEADDR" to
`tests: net: sockets: add tests for SO_REUSEADDR`